pkgs/applications/networking/instant-messengers/caprine-bin/build-from-appimage.nix

```diff
@@ -27,6 +28,10 @@ in
   '';
 
   extraInstallCommands = ''
+    source "${makeWrapper}/nix-support/setup-hook"
```

Comment:
`wrapType2` has `nativeBuildInputs` now so use it instead. https://github.com/NixOS/nixpkgs/pull/344516